cellpack/autopack/Analysis.py

```diff
@@ -1179,7 +1179,7 @@ def doloop(
             across seeds
         """
         number_of_packings = packing_config_data["number_of_packings"]
-        plot_figures = packing_config_data.get("save_plot_figures", True)
+        plot_figures = packing_config_data["save_plot_figures"]
```

Comment:
what's the reason for this change?

Reply:
We added a default entry for `save_plot_figures` in `config_loader.py` (it did not exist before). The pattern now matches how other settings are updated from the config.